src/requestCertificate.scss

```diff
@@ -5,3 +5,8 @@
 .storage-row > * {
   margin-right: 1.5rem;
 }
+
+.dns-helper-text {
```

Comment:
This is because the textarea and the input doesn't quite work the same and it gives you a little extra gap, compared to the [example that uses input](https://www.patternfly.org/v4/documentation/react/components/form/basic), right?
Would be good to file this as a Patternfly issue, so we don't have to reimplement this for any helper text for textareas.

Reply:
Yes Patternfly's FormGroup adds too much padding at top which makes it seems too far away from it.

Reply:
Now I'm thinking, is this really Patternfly issue? Defaultly in patternfly textareas and inputs are spaced much futher away from each other so helper-text fits is spaced fine too, but we change that in `pkg/lib/form-layout.scss` by assinging `grid-grep: 0.5rem 1.5rem;` to ct-from class, so I guess we also have to make the gap small for helper texts too, no?

Reply:
I'm working on ct-form right now to address PF4 issues; both with better PF4 support and also to make sure it's using _current_ PF4-specified spacing (as it has updated a few times over the past year). I'm hoping to have something soon you can update to, so we can address stuff like this.

Reply:
(But don't visual spacing and such block the PR.)

Reply:
Oh okay. I can create an issue to update ct-form later when it's updated in cockpit repo. So I guess this is not a issue for this PR

Reply:
https://github.com/skobyda/cockpit-certificates/issues/21

Reply:
Working on addressing spacing (and more) @ https://github.com/cockpit-project/cockpit/pull/14306

Reply:
Sorry, I should tried this earlier with, but new ct-form doesn't change the spacing for helper text. When I delete this custom dns-helper-text class, the spacing looks like this:
![Screenshot from 2020-07-09 15-40-43](https://user-images.githubusercontent.com/42733240/87047518-ea977280-c1fa-11ea-8fbb-a3d413541634.png)

@garrett Can something be changed in ct-form to address this?